code/src/java/pcgen/core/term/PCBaseHDTermEvaluator.java

```diff
@@ -73,7 +73,7 @@ public boolean isSourceDependant()
 	/* (non-Javadoc)
 	 * @see pcgen.core.term.TermEvaluator#isStatic()
 	 */
-	public boolean isStatic()
+	public static boolean isStatic()
```

Comment:
This is an example of where automation scares me.  This was made static, but was once part of the TermEvaluator interface.  I'd rather we have a discussion of whether ALL the isStatic() calls can be removed rather than converting them to static and then it being needed again and having to undo all the changes.